.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,22 @@
+name: Feature Request
+description: Create a feature request for Quilt Standard Libraries.
+labels: [suggestion]
+body:
+  - type: markdown
+    attributes:
+      value: "## Welcome!"
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to create your feature suggestion! If you want to discuss your feature request, join us on [Discord](https://discord.quiltmc.org).
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe how your feature(s) will work if applicable.
+      description: "Describe the suggestion as clearly as possible. More info means your feature request is more likely to be accepted."
+
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional context
+      description: "Add any other context about your feature request here."
```

Comment:
Refocus this as an "API Request" and also make there be a field for which library and possibly module the new API should go into